inst/ms/aggreCAT.qmd

```diff
@@ -596,62 +625,81 @@ Below we demonstrate the application of `ArMean` on a single claim
 #| results: hold
 #| echo: false
-knitr::include_graphics(path = "images/paste-80B1C5CB.png")
+knitr::include_graphics(path = "images/ArMean.png")
 ```
 
-::: {#aggWorkflow .callout}
-#### Box 1: Aggregation Workflow Blueprint
-
--   wrapper functions and the 'type' argument
-
--   default arg structure for each wrapper fun
-
-    -   percent toggle
-
-    -   placeholder
-
-    -   name
-
-    -   What else?
+::: {.callout}
+:::{#aggWorkflow}
+### Box 1: Aggregation Workflow Blueprint{.unnumbered}
 
--   COmputation:
+#### Argument Structure and Expected Form
 
-    -   Weighting Functions
-
-    -   Note that not all functions use a separately defined weighting
-        function -- for simpler weight computations, these are defined
-        in-function rather than being modularised [See Table
-        1](#table1).
-
-Each aggregation function follow a general workflow whereby the primary
-dataset `data_ratings`, parsed to the `expert_judgements` argument, is
-first pre-processed by `pre_process_judgements()`, subsequently the
-aggregation method is applied using `dplyr::summarise()`, and then
-finally the aggregated data is parsed to `postprocess_judgements()`.
-This general aggregation method workflow is seen best in `ArMean()`:
+Each aggregation *wrapper* function takes the following arguments:
+`expert_judgements`, `type`, `name`, `placeholder` and `percent_toggle`:
 
 ```{r}
-#| label: blah
-AverageWAgg
+#| prompt: true
+args(AverageWAgg)
 ```
 
+The aggregation *method* to be applied by the aggregation *function*, is
+specified by the `type` argument, defaulting to `ArMean` in the above example. The resultant
+`tibble` of Confidence Scores includes the `name` of the aggregation
+method applied, defaulting to the `type` argument, but this can be
+overridden by the user if they supply a non-`NULL` value to `name`.
+
+Aggregation functions expect the judgements supplied in
+`expert_judgements` to be percentage chances. However, oftentimes
+analysts may wish to elicit probabilities, counts, or other numeric
+quantities. By setting `percent_toggle` to `TRUE`, the user can turn
+toggle the expectation of percentage values *off*, providing
+non-percentage numeric judgements..
+
+When working with regularly updated data and developing a reproducible
+pipeline [@Yenni2019] , it can be useful to put aggregation methods into
+'placeholder' mode, whereby a placeholder value is returned by he
+aggregation function instead of computing a Confidence Score using the
+aggregation method. By setting `placeholder` to `TRUE`, the user can
+supply a placeholder Confidence Score, which defaults to $65\%$, the
+average replicability of SBS research (citation? Fiona/David?). Should
```

Comment:
That number choice predates my joining the project, so Fiona would be the best to ask for a citation. Might even by in one of the slides Fiona uses to introduce the project.

Reply:
- [ ] Elliot to message Fiona
- [ ] Elliot to update MS once answered